azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/implementation/AvailabilitySetImpl.java

```diff
@@ -101,28 +101,32 @@ public ServiceCall applyAsync(ServiceCallback<AvailabilitySet> callback) {
         return this.createAsync(callback);
     }
 
+    // CreatorTaskGroup.ResourceCreator implementation
+
     @Override
-    protected void createResource() throws Exception {
+    public Resource createResource() throws Exception {
```

Comment:
we are trying to simplfy the logic we had earleier by removing two PUBLIC methods `createRootResource` and `createRootResourceAsync` exposed in every resource (via CreatableImpl inheritance). These two PUBLIC methods wraps calls to PROTECTED `createResource` and `createResourceAsync` methods. Removing createRoot* methods caused making the methods its wraps to public.